.github/workflows/test-gaia-v14-stateful-upgrade-steps.yml

```
@@ -0,0 +1,359 @@
+      - name: Initialize consumer chain b with ICS v2.0.0
+        env:
+          CONSUMER_CHAIN_ID: consumerb
+          CONSUMER_HOME_1: /home/runner/.conb1
+          CON1_API_PORT: 25201
+          CON1_GRPC_PORT: 26201
+          CON1_RPC_PORT: 27201
+          CON1_P2P_PORT: 28201
+          CON1_PPROF_PORT: 6165
+          CONSUMER_SERVICE_1: conb1.service
+          CONSUMER_CHAIN_BINARY: ${{ vars.CONSUMER_CHAIN_BINARY_2 }}
+          DOWNTIME_BLOCKS: 1000
+        run: tests/major_stateful_upgrade/init_consumer.sh
+      - name: Launch consumer chain b
+        env:
+          CONSUMER_CHAIN_ID: consumerb
+          CONSUMER_HOME_1: /home/runner/.conb1
+          CONSUMER_SERVICE_1: conb1.service
+        run: tests/major_stateful_upgrade/launch_consumer.sh
+      - name: _Block Production> ICS v2.0.0
+        run: tests/test_block_production.sh 127.0.0.1 27201 10
+      - name: Establish CCV channel for consumer b
+        run: |
+          echo "Stopping hermes"
+          killall hermes || true
+          screen -XS hermes.service quit || true
+          client_id=$(gaiad --home $HOME/.val1 q provider list-consumer-chains --output json | jq -r '.chains[] | select(.chain_id=="consumerb").client_id')
+          hermes create connection --a-chain consumerb --a-client 07-tendermint-0 --b-client $client_id
+          hermes create channel --a-chain consumerb --a-port consumer --b-port provider --order ordered --a-connection connection-0 --channel-version 1
+          echo "Starting Hermes"
+          screen -L -Logfile $HOME/artifact/hermes.service.log -S hermes.service -d -m bash $HOME/hermes.service.sh
+          # set screen to flush log to 0
+          screen -r hermes.service -p0 -X logfile flush 0
+          echo "Wait for hermes to start..."
+          sleep 180
+      - name: _VSC successful> ICS v2.0.0
+        env:
+          CON1_RPC_PORT: 27201
+        run: tests/major_stateful_upgrade/test_ccv.sh
+      - name: _IBC transfer successful> ICS v2.0.0
+        env:
+          CONSUMER_CHAIN_ID: consumerb
+          CONSUMER_HOME_1: /home/runner/.conb1
+        run: |
+          client_id=$(gaiad --home $HOME/.val1 q provider list-consumer-chains --output json | jq -r '.chains[] | select(.chain_id=="consumerb").client_id')
+          echo "Client ID: $client_id"
+          connection_id=$(hermes --json query client connections --chain local-testnet --client $client_id | jq -r '. | select(.result).result[]')
+          echo "Connection ID: $connection_id"
+          channel_id=$(hermes --json query connection channels --chain local-testnet --connection $connection_id | jq -r '. | select(.result).result[] | select(.port_id=="transfer").channel_id')
+          echo "Channel ID: $channel_id"
+          tests/major_stateful_upgrade/test_consumer_ibc_transfer.sh $channel_id 3
+      - name: Start Stride chain
+        run: tests/major_stateful_upgrade/start_stride.sh
+      - name: _Pre-upgrade block Production> Stride
```

```suggestion
      - name: Pre-upgrade block Production> Stride
```
We can skip showing this one